tests/service/infoService.spec.js

```diff
@@ -0,0 +1,20 @@
+describe( 'InfoService', function() {
+    var InfoService, uibModal;
+
+    beforeEach( module( 'SolrHeatmapApp' ) );
+
+    beforeEach( inject( function( _InfoService_, _$uibModal_) {
+
+        InfoService = _InfoService_;
+        uibModal = _$uibModal_;
+    }));
+
+    describe('#showInfo', function() {
+        it('opens the modal info', function() {
+            pending();
```

Comment:
replace with:
`solrHeatmapApp.instructions = {'textsearch': 'Test'};`

solrHeatmapApp exists but the text is loaded form a json, we need to simulate this until we have the config service